### PR TITLE
Complete Core skeleton

### PR DESCRIPTION
**Pull request by dimitri-yatsenko**

### CODE REVIEW

File: src/core/index.md [hunk 1]
```diff
@@ -1,63 +1,61 @@
...
--   :fontawesome-brands-python:{ .lg .middle } **DataJoint Python**
+-   :fontawesome-brands-python:{ .lg .middle } **DataJoint for Python**
 
     ---
 
-    A low-level client for managing [data pipelines](./glossary#data-pipeline).
+    A Python client for defining, operating, and querying data pipelines.
 
-    :octicons-arrow-right-24: New docs coming soon!  In the meantime, refer to our [legacy docs](https://docs.datajoint.org/python/)
+    :octicons-arrow-right-24: [Documentation](https://docs.datajoint.org/python/) | [Source code](https://datajoint.com/datajoint/datajoint-matlab)
```
Comment (CBroz1) [inline src/core/index.md:15]: Below updating 
1. To point to new mkdocs
2. matlab -> python. I assume by 'source code', we mean github, right? 
```suggestion
    :octicons-arrow-right-24: [Documentation](https://datajoint.com/docs/core/datajoint-python/) | [Source code](https://github.com/datajoint/datajoint-python)
```

File: src/core/index.md [hunk 2]
```diff
@@ -1,63 +1,61 @@
...
--   :fontawesome-brands-java:{ .lg .middle } **DataJoint MATLAB**
+-   :fontawesome-brands-java:{ .lg .middle } **DataJoint for MATLAB**
 
     ---
 
-    A low-level client for managing [data pipelines](./glossary#data-pipeline).
+    A MATLAB client for defining, operating, and querying data pipelines.
 
-    :octicons-arrow-right-24: New docs coming soon!  In the meantime, refer to our [legacy docs](https://docs.datajoint.org/matlab/).
+    :octicons-arrow-right-24: [Documentation](https://docs.datajoint.org/matlab/) | [Source code](https://datajoint.com/datajoint/datajoint-matlab)
```
Comment (CBroz1) [inline src/core/index.md:23]:
```suggestion
    :octicons-arrow-right-24: [Documentation](https://datajoint.com/docs/core/datajoint-matlab) | [Source code](https://github.com/datajoint/datajoint-matlab)
```

File: src/elements/concepts.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ reference the database schema.
 An Element is a software package defining one or more DataJoint schemas serving a
 particular purpose. By convention, such packages are hosted in individual GitHub
 repositories. For example, Element `element_calcium_imaging` is hosted at
-https://github.com/datajoint/element-calcium-imaging and contains two DataJoint
+[GitHub Repo](https://github.com/datajoint/element-calcium-imaging) and contains two DataJoint
```
Comment (CBroz1) [inline src/elements/concepts.md:40]:
```suggestion
[this GitHub repository](https://github.com/datajoint/element-calcium-imaging) and contains two DataJoint
```

File: src/index.md [hunk 4]
```diff
@@ -36,9 +36,7 @@
 
      ---
 
-     Managed services for neurophysiology built with DataJoint Core and DataJoint Elements.
-
-     [:octicons-arrow-right-24: Learn more](./works/)
+     https://works.datajoint.com:  A cloud platform for automated analysis workflows. It relies on DataJoint Core and DataJoint Elements.
```
Comment (CBroz1) [inline src/index.md:39]:
```suggestion
     [:octicons-arrow-right-24: Try it out](https://works.datajoint.com):  A cloud platform for automated analysis workflows. It relies on DataJoint Core and DataJoint Elements.
```

File: src/projects.md [hunk 5]
```diff
@@ -89,6 +89,7 @@ of Medicine to meet the needs of their own research. Below is a partial list of
   + <a href="http://pblab.tau.ac.il/en/" target="_blank">Pablo Blinder Lab</a> (<a href="https://github.com/PBLab" target="_blank">GitHub</a>)
 + University of Bonn
   + <a href="https://rose-group.ieecr-bonn.de" target="_blank">Tobias Rose Lab</a>
+  + <a href="https://https://www.ukbonn.de/en/epileptology/workgroups/mormann-workgroup-cognitive-und-clinical-neurophysiology/">Mormann Workgroup</a>
```
Comment (CBroz1) [inline src/projects.md:92]: Please also update the the `<!-- numbers -->` in headings above to update our running count.

File: src/core/concepts/glossary.md [hunk 6]
```diff
@@ -1,16 +1,15 @@
...
+We've taken careful consideration to use consistent terminology. 
 
 <!-- Contributors: Please keep this table in alphabetical order -->
 
 | Term | Definition |
 | --- | --- |
-| <span id="DAG">DAG</span> | directed acyclic graph (DAG) is a set of nodes and unidirectional edges with no cyclic dependencies. No child node points to a parent node. |
-| <span id="data-pipeline">data pipeline</span> | formal definition of a directed acyclic graph (DAG) of processes that achieves the [DataJoint Mantra](../concepts/mantra) |
+| <span id="DAG">DAG</span> | directed acyclic graph (DAG) is a set of nodes and connected with a set of directed edges that form no cycles. This means that there is never a path back to a node after passing through it by following the directed edges. Formal workflow management systems represent workflows in the form of DAGs. |
+| <span id="data-pipeline">data pipeline</span> | A sequence of data transformation steps from data sources through multiple intermediate structures. More generally, a data pipeline is a directed acyclic graph.  In DataJoint, each step is represented by a table in a relational database. |
```
Comment (guzman-raphael) [inline src/core/concepts/glossary.md:12]: Should we mention compute somewhere here at all?

File: src/core/index.md [hunk 7]
```diff
@@ -1,63 +1,61 @@
 # Core
 
-DataJoint Core projects are fully open-source and are built to develop, define, manage,
-and visualize [data pipelines](./getting-started/data-pipelines). Below are the projects that
-make up the family of core open-source projects.
+DataJoint Core is a collection of open-source tools for building and operating scientific data pipelines.
```
Comment (guzman-raphael) [inline src/core/index.md:3]: For the sake of SEO, we should make sure to link back to glossary and between pages as much as possible.

```suggestion
DataJoint Core is a collection of open-source tools for building, operating, and visualizing scientific [data pipelines](./concepts/glossary#data-pipeline).
```

File: mkdocs.yaml [hunk 8]
```diff
@@ -8,45 +8,72 @@ nav:
...
-        - Make Methods: core/concepts/reproduce/make-method.md
-      - Referential Integrity:
-        - Query Backend: core/concepts/ref-integrity/query-backend.md
-        - Distributed Computing: core/concepts/ref-integrity/distributed-computing.md
-        - Relational Databases: core/concepts/ref-integrity/relational-databases.md
-      - Glossary: core/glossary.md
-      - FAQ: core/faq.md
+    - Concepts: 
+      - Principles: core/concepts/principles.md
+      - Glossary: core/concepts/glossary.md
```
Comment (guzman-raphael) [inline mkdocs.yaml:13]: Why not have the glossary to be further out? This seems a bit buried. For example, I normally review the glossary first to establish terms before reading deeper in the documentation of other projects.

File: src/core/index.md [hunk 9]
```diff
@@ -1,63 +1,61 @@
...
 
 <div class="grid cards" markdown>
 
--   :fontawesome-brands-python:{ .lg .middle } **DataJoint Python**
+-   :fontawesome-brands-python:{ .lg .middle } **DataJoint for Python**
 
     ---
 
-    A low-level client for managing [data pipelines](./glossary#data-pipeline).
+    A Python client for defining, operating, and querying data pipelines.
```
Comment (guzman-raphael) [inline src/core/index.md:13]: Not sure I understand the purpose of mentioning Python here multiple times. It is in the title of the card. Why do we need to repeat this? Also, we should like to the glossary the usage of `data pipelines`.

File: src/core/index.md [hunk 10]
```diff
@@ -1,63 +1,61 @@
...
-    :octicons-arrow-right-24: New docs coming soon!  In the meantime, refer to our [legacy docs](https://docs.datajoint.org/python/)
+    :octicons-arrow-right-24: [Documentation](https://docs.datajoint.org/python/) | [Source code](https://datajoint.com/datajoint/datajoint-matlab)
 
--   :fontawesome-brands-java:{ .lg .middle } **DataJoint MATLAB**
+-   :fontawesome-brands-java:{ .lg .middle } **DataJoint for MATLAB**
 
     ---
 
-    A low-level client for managing [data pipelines](./glossary#data-pipeline).
+    A MATLAB client for defining, operating, and querying data pipelines.
```
Comment (guzman-raphael) [inline src/core/index.md:21]: Similar to previous comment.

File: src/core/index.md [hunk 11]
```diff
@@ -1,63 +1,61 @@
...
-
-    :octicons-arrow-right-24: New docs coming soon!  In the meantime, refer to our [legacy docs](./pharus/).
+    A REST API server for interacting with DataJoint pipelines.
 
+    :octicons-arrow-right-24: [Documentation](https://datajoint.github.io/pharus/) | [Source code](https://github.com/datajoint/pharus/)
+ 
 </div>
 
-## Web GUIs
+## Web Applicaitons
```
Comment (guzman-raphael) [inline src/core/index.md:35]:
```suggestion
## Web Applications
```

File: src/core/index.md [hunk 12]
```diff
@@ -1,63 +1,61 @@
...
-    :octicons-arrow-right-24: New docs coming soon!  In the meantime, refer to our [legacy docs](./datajoint-labbook/).
+    :octicons-arrow-right-24: [Documentation](https://datajoint.github.io/datajoint-labbook/) | [Source code](https://github.com/datajoint/datajoint-labbook/)
 
 -   :fontawesome-brands-chrome:{ .lg .middle } **SciViz**
 
     ---
 
-    A visualization framework for making low-code web apps for 
-    [data pipelines](./glossary#data-pipeline).
+    A framework for making low-code web apps for data visualization.
```
Comment (guzman-raphael) [inline src/core/index.md:52]:
```suggestion
    A framework for making low-code web apps for [data pipelines](./concepts/glossary#data-pipeline).
```

File: src/core/index.md [hunk 13]
```diff
@@ -1,63 +1,61 @@
...
 
-    :octicons-arrow-right-24: New docs coming soon!
+    :octicons-arrow-right-24: [Source code](https://github.com/datajoint/sci-viz)
 
 </div>
 
-## Container Images
+## Docker Images
```
Comment (guzman-raphael) [inline src/core/index.md:58]: Docker is a particular technology that provides a solution for containerizing workloads. We may continue using Docker or change it in the future. The more appropriate term here is container.

```suggestion
## Container Images
```

File: src/core/index.md [hunk 14]
```diff
@@ -77,63 +75,49 @@ graph
...
 
     ---
+    MySQL server pre-configured to work smoothly with DataJoint. 
 
-    An optimized, MySQL backend for [data pipelines](./glossary#data-pipeline).
-
-    :octicons-arrow-right-24: New docs coming soon!
-
--   :fontawesome-brands-docker:{ .lg .middle } **datajoint/miniconda3**
+-   :fontawesome-brands-docker:{ .lg .middle } [**datajoint/miniconda3**](https://hub.docker.com/r/datajoint/niniconda3)
```
Comment (guzman-raphael) [inline src/core/index.md:85]:
```suggestion
-   :fontawesome-brands-docker:{ .lg .middle } [**datajoint/miniconda3**](https://hub.docker.com/r/datajoint/miniconda3)
```

File: src/elements/concepts.md (hunk 3 above)
Comment (guzman-raphael) [inline src/elements/concepts.md:40]: Perhaps instead:

```
For example, Element [element_calcium_imaging](https://github.com/datajoint/element-calcium-imaging) contains two DataJoint
```

**Review by guzman-raphael — CHANGES_REQUESTED** [review]
Thanks for the help @dimitri-yatsenko! :muscle:

Had some thoughts on improvement while reviewing this.

File: mkdocs.yaml [hunk 15]
```diff
@@ -8,45 +8,72 @@ nav:
...
+      - Schema Diagrams: core/design/diagrams.md
+      - Entity Normalization: core/design/normalization.md 
+      - Data Integrity: core/design/integrity.md
+      - Schema Recall: core/design/recall.md
+      - Schema Drop: core/design/drop.md
+      - Schema Modification: core/design/alter.md
+    - Data Manipulations:
+      - Insert: core/manip/insert.md
+      - Delete: core/manip/delete.md
+      - Update: core/manip/update.md
```
[the remark this answers is not in the document]
Comment (dimitri-yatsenko) [reply mkdocs.yaml:46]: `ALTER` is part of DDL -- it changes the schema definition. Insert, delete, update are part of DML.

File: src/core/concepts/principles.md [hunk 16]
```diff
@@ -1,4 +1,4 @@
-# Mantra
+# Key Principes
 
 The *DataJoint Mantra* consists of three main objectives:
```
[the remark this answers is not in the document]
Comment (dimitri-yatsenko) [reply src/core/concepts/principles.md:3]: This section will be completely reworked in the next commit.

File: src/index.md [hunk 17]
```diff
@@ -1,59 +1,26 @@
 # Welcome to the DataJoint Documentation
 
-<h2> DataJoint Ecosystem </h2>
+## DataJoint Core
```
Comment (CBroz1) [inline src/index.md:3]: What motivated the choice to remove cards? The [elements doc page](https://datajoint.com/docs/elements/) takes this approach. I'm wondering if we should take the same approach there

**Review by guzman-raphael — APPROVED** [review]
Thanks @dimitri-yatsenko. Will merge as-is and move suggestions into new PR"s to keep it easier to manage reviews.

File: src/projects.md (hunk 5 above)
Comment (guzman-raphael) [reply src/projects.md:92]: TBD in subsequent PR's.

File: src/core/concepts/principles.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (guzman-raphael) [reply src/core/concepts/principles.md]: TBD in subsequent PR's.

File: mkdocs.yaml (hunk 15 above)
[the remark this answers is not in the document]
Comment (guzman-raphael) [reply mkdocs.yaml]: TBD in subsequent PR's.

File: mkdocs.yaml [hunk 18]
```diff
@@ -8,45 +8,72 @@ nav:
...
+      - File Storage: core/sysadmin/filestore.md
+    - Client Configuration: 
+      - Install: core/client/install.md
+      - Credentials: core/client/creds.md
+      - Settings: core/client/settings.md
+      - File stores: core/client/stores.md
+    - Schema Design:
+      - Schema Creation: core/design/schema.md
+      - Table Definition:
+        - Table tiers: core/design/tiers.md
```
[the remark this answers is not in the document]
Comment (guzman-raphael) [reply mkdocs.yaml]: TBD in subsequent PR's.

File: mkdocs.yaml [hunk 19]
```diff
@@ -8,45 +8,72 @@ nav:
...
+      - Principles: core/concepts/principles.md
+      - Glossary: core/concepts/glossary.md
+    - System Administration: 
+      - Database Administration: core/sysadmin/dba.md
+      - File Storage: core/sysadmin/filestore.md
+    - Client Configuration: 
+      - Install: core/client/install.md
+      - Credentials: core/client/creds.md
+      - Settings: core/client/settings.md
+      - File stores: core/client/stores.md
```
[the remark this answers is not in the document]
Comment (guzman-raphael) [reply mkdocs.yaml:21]: TBD in subsequent PR's.

File: mkdocs.yaml [hunk 20]
```diff
@@ -8,45 +8,72 @@ nav:
...
+        - Custom Datatypes: core/design/customtype.md
+        - Dependencies: core/design/dependencies.md
+        - Indexes: core/design/indexes.md
+        - Master-Part Relationships: core/design/master-part.md
+      - Schema Diagrams: core/design/diagrams.md
+      - Entity Normalization: core/design/normalization.md 
+      - Data Integrity: core/design/integrity.md
+      - Schema Recall: core/design/recall.md
+      - Schema Drop: core/design/drop.md
+      - Schema Modification: core/design/alter.md
```
[the remark this answers is not in the document]
Comment (guzman-raphael) [reply mkdocs.yaml:42]: TBD in subsequent PR's.

File: mkdocs.yaml [hunk 21]
```diff
@@ -8,45 +8,72 @@ nav:
...
+    - Data Manipulations:
+      - Insert: core/manip/insert.md
+      - Delete: core/manip/delete.md
+      - Update: core/manip/update.md
+      - Transactions: core/manip/transactions.md
+    - Data queries:
+      - Query principles: core/query/operators.md
```
[the remark this answers is not in the document]
Comment (guzman-raphael) [reply mkdocs.yaml:49]: TBD in subsequent PR's.

File: src/core/client/creds.md [hunk 22]
```diff
@@ -0,0 +1,3 @@
+## Work in progress
+You may ask questions through the chatbot below or
+refer to legacy documentation: https://https://docs.datajoint.org/
```
[the remark this answers is not in the document]
Comment (guzman-raphael) [reply src/core/client/creds.md]: TBD in subsequent PR's.

File: src/core/concepts/glossary.md (hunk 6 above)
Comment (guzman-raphael) [reply src/core/concepts/glossary.md:12]: TBD in subsequent PR's.

File: src/core/index.md (hunk 7 above)
Comment (guzman-raphael) [reply src/core/index.md:3]: TBD in subsequent PR's.

File: src/core/index.md (hunk 9 above)
Comment (guzman-raphael) [reply src/core/index.md:13]: TBD in subsequent PR's.

File: src/core/index.md (hunk 10 above)
Comment (guzman-raphael) [reply src/core/index.md:23]: TBD in subsequent PR's.

File: mkdocs.yaml (hunk 8 above)
Comment (guzman-raphael) [reply mkdocs.yaml:13]: TBD in subsequent PR's.

File: src/core/index.md (hunk 12 above)
Comment (guzman-raphael) [reply src/core/index.md:56]: TBD in subsequent PR's.

File: src/elements/concepts.md (hunk 3 above)
Comment (guzman-raphael) [reply src/elements/concepts.md]: TBD in subsequent PR's.

File: src/index.md (hunk 17 above)
Comment (guzman-raphael) [reply src/index.md]: TBD in subsequent PR's.

**Pull request merged**